src/Microsoft.ML.Core/Utilities/Utils.cs

```diff
@@ -534,15 +534,7 @@ public static int[] GetRandomPermutation(Random rand, int size)
             return res;
         }
 
-        public static void Shuffle<T>(Random rand, Span<T> rgv)
-        {
-            Contracts.AssertValue(rand);
-
-            for (int iv = 0; iv < rgv.Length; iv++)
-                Swap(ref rgv[iv], ref rgv[iv + rand.Next(rgv.Length - iv)]);
-        }
-
-        public static bool AreEqual(float[] arr1, float[] arr2)
+        public static bool AreEqual(Single[] arr1, Single[] arr2)
```

Comment:
>Single [](start = 36, length = 6)

why not float?

Reply:
Woah, good catch. Weird merge reversion.